modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -1,75 +1,81 @@
 = AS2 Receive Endpoint Settings
 
-AS2 endpoint settings specify where to receive inbound files from your trading partners. You can use the same AS2 receive endpoints to receive inbound transmissions from multiple partners.
+AS2 endpoint settings dictate where to receive inbound files from your trading partners. You can use the same AS2 receive endpoints to receive inbound transmissions from multiple partners.
 
-Create the AS2 identifiers for your organization and your trading partners in the respective host and trading partner profile pages.
+You can create the AS2 identifiers for your organization and for your trading partners in the respective xref:configure-host.adoc[host] and xref:configure-partner.adoc[trading partner] profile pages.
 
 [%header%autowidth.spread]
 |===
-|Setting |Description | Required
-|Endpoint Name
-|Name used to easily identify the purpose of this endpoint. The name is auto-populated, but can be overwritten. 
-|Yes 
+| Setting | Description | Required
+| Endpoint Name
+| The name used to identify the endpoint. The `Endpoint Name` is automatically generated, but can be modified manually.
```

Comment:
The automatically generated name used to identify the endpoint, which you can subsequently modify manually.

Reply:
Updated all three topics to reflect this change.